notes/ltc-transfer-penalty.vbs

```diff
@@ -140,71 +137,90 @@ IF transfer_amount <> "" THEN transfer_amount = "$" & transfer_amount
-		Dialog Dialog1
-		cancel_confirmation 	'asks user if they really want to cancel.  If yes, then script stops.  If no, loops back to dialog.
-		If (len(baseline_date) < 8 or IsDate(baseline_date) = FALSE) then err_msg = err_msg & vbNewLine & "You must enter a valid baseline date in the MM/DD/YYYY format.  See policy reference at the bottom of the dialog box if you are unsure of how to determined the baseline date."
-		If worker_signature = "" then err_msg = err_msg & vbNewLine & "You must sign your case note!"
-		If type_of_transfer_list = "Select one..." then err_msg = err_msg & vbNewLine & "You must select the type of transfer."
-		If (type_of_transfer_list = "Other" AND other_information = "") then err_msg = err_msg & vbNewLine & "You have selected ""Other"" as your transfer reason.  You must complete the 'other transfer info' field."
-		If (transfer_date = "" or IsDate(transfer_date) = FALSE) then err_msg = err_msg & vbNewLine & "You must enter a valid transfer date."
-		If (date_of_application = "" or IsDate(date_of_application) = FALSE) then err_msg = err_msg & vbNewLine & "You must enter a valid application date."
-		If (transfer_amount = "" or IsNumeric(transfer_amount) = FALSE) then err_msg = err_msg & vbNewLine & "You must enter a valid transfer penalty amount."
-		If (date_client_was_otherwise_eligible = "" or IsDate(date_client_was_otherwise_eligible) = FALSE) then err_msg = err_msg & vbNewLine & "You must enter a valid date that the client was otherwise eligible for MA."
-		If period_begins = "" then err_msg = err_msg & vbNewLine & "You must enter the start date of the transfer penalty."
-		If last_full_month_of_period = "" then err_msg = err_msg & vbNewLine & "You must enter the last full month of the transfer penalty."
-		If (partial_penalty_amount = "" or IsNumeric(partial_penalty_amount) = False) then err_msg = err_msg & vbNewLine & "You must enter the partial penalty amount, even if the amount is 0."
-	    If case_action = "" then err_msg = err_msg & vbNewLine & "You must case note the case action."
-        If err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbNewLine & err_msg & vbNewLine		'error message including instruction on what needs to be fixed from each mandatory field if incorrect
+	  DO
+        err_msg = ""
+        Dialog1 = "" 'Blanking out previous dialog detail
+        BeginDialog Dialog1, 0, 0, 331, 270, "LTC Transfer Penalty Information"
+          EditBox 65, 45, 60, 15, baseline_date
+          EditBox 250, 45, 60, 15, period_begins
+          EditBox 65, 65, 60, 15, transfer_date
+          EditBox 250, 65, 60, 15, last_full_month_of_period
+          EditBox 65, 85, 60, 15, transfer_amount
+          EditBox 250, 85, 60, 15, partial_penalty_amount
+          EditBox 125, 115, 60, 15, date_of_application
+          EditBox 125, 135, 60, 15, date_client_was_otherwise_eligible
+          DropListBox 125, 155, 60, 15, "Select one..."+chr(9)+"Approved"+chr(9)+"Denied"+chr(9)+"Requested", waiver_list
```

Comment:
Let's add a N/A option to waiver_list. It's possible that a resident doesn't request a hardship and just takes the penalty.

Reply:
Added N/A to the dropdown and removed field requirement for hardship_waiver_details. Let me know if you feel this should be required considering the addition of the N/A option for the dropdown.